src/fu-util.c

```diff
@@ -2564,14 +2611,14 @@ fu_util_update(FuUtilPrivate *priv, gchar **values, GError **error)
 						 &error_local);
 		if (rels == NULL) {
 			if (!latest_header) {
-				g_printerr(
-				    "%s\n",
+				fu_console_println_literal(
+				    priv->console,
```

Comment:
this used to be on stderr but is now on stdout right?  Is that intended?

Reply:
I guess I'm torn on what we should do with stderr -- on one hand if we're doing anything thats going to be captured or shown in logs then using --json is probably strongly advised. On the other hand it gets a bit handwavy on what goes to stderr and what goes to stdout. Perhaps a better API would be to have:

 * fu_console_print_literal() -- with a newline
 * fu_console_print() -- with a newline

for the common case, and then have a

 * fu_console_print_literal_full()
 * fu_console_print_full()

With FU_CONSOLE_PRINT_FLAG_NEWLINE and FU_CONSOLE_PRINT_FLAG_STDERR

That's probably going to cover all usecases.

Reply:
I did that, and it seems to work okay.